articles/140_topic_and_service_name_mapping.md

```diff
@@ -298,35 +251,29 @@ Therefore to allow ROS programs to interoperate with "native" DDS topic names th
 #### Examples and Ideas for Communicating with Non-ROS Topics
 
 Note that these are not part of the proposal, but only possible solutions to the issue of communicating with "native" DDS topics.
-This section should be update when/if a complete solution is found.
+This section should be updated when/if a complete solution is found.
```

Comment:
I think you can actually update the design doc here as well, since we have the option to ignore the ROS conventions.

Reply:
Now that we can use `avoid_ros_namespace_convention` Should I remove the alternatives of using the 'to have some markup in the scheme name' in the same section below? or leave it as is?

Reply:
Having a markup to express the exact topic name is separate from having the ability to avoid applying our ROS specific mangling in our API. So I think it should _not_ be removed because we have `avoid_ros_namespace_convention`.

Reply:
Thanks, I'll have it listed as an alternative in the same section